src/app/connections/list/list.component.html

```diff
@@ -1,47 +1,8 @@
 <ipaas-loading [loading]="loading">
-  <div class="container-cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-xs-12 col-sm-3 col-md-2 card" *ngFor="let connection of connections">
-        <div class="card-pf card-pf-view card-pf-view-xs">
-          <div class="card-pf-body">
-
-            <!-- Card Dropdown -->
-
-            <div class="card-pf-heading-kebab">
-              <div class="dropdown pull-right dropdown-kebab-pf card-pf-time-frame-filter">
-                <button type="button" class="btn btn-link dropdown-toggle" data-toggle="dropdown">
-                  <span class="fa fa-ellipsis-v"></span>
-                </button>
-                <ul class="dropdown-menu dropdown-menu-right" role="menu">
-                  <li>
-                    <a [routerLink]="[connection.id]">View</a>
-                  </li>
-                  <li>
-                    <a [routerLink]="[connection.id, 'edit']">Edit</a>
-                  </li>
-                  <li>
-                    <a>Duplicate</a>
-                  </li>
-                  <li>
-                    <a>Delete</a>
-                  </li>
-                </ul>
-              </div>
-
-              <!-- Card Title & Icon -->
-
-              <h2 class="card-pf-title">
-                <span class="fa {{ connection.icon }}"></span>
-                <a [routerLink]="[connection.id]" class="card-title">{{ connection.name }}</a>
-              </h2>
-            </div>
-
-            <!-- Card Description -->
-
-            <p>{{ connection.description | truncate : truncateLimit : truncateTrail }}</p>
-          </div>
-        </div>
-      </div>
-    </div>
+  <div class='connections list'>
+    <ipaas-connections-list-toolbar [childConnections]="connections | async"
+                                    (change)="updateConnections($event)"></ipaas-connections-list-toolbar>
+    <ipaas-connections-list-page [connectionList]="connections"
+                                 [loading]="loading"></ipaas-connections-list-page>
```

Comment:
This is effectively swapping the list-page and the list components, but I don't understand why.  The way I understood the previous approach is the page handles loading the data from the store, and passes it along to the list/toolbar views to render/filter appropriately.  Now it looks like the `page` component is loading and rendering

Note, I have a vested interest in this as I'm re-using the `ipaas-connections-list` and `ipaas-connections-list-toolbar` components in the integration editor.  :-)